src/Forte.Optimizely.ContentUsage/Frontend/Lib/ContentUsageAPIClient.ts

```diff
@@ -62,7 +62,10 @@ export default class ContentUsageAPIClient {
     return this.get<ContentTypeDto>(this.endpoints.contentType, { guid });
   }
 
-  public async getContentTypeUsages(guid: string) {
-    return this.get<ContentUsageDto[]>(this.endpoints.contentUsages, { guid });
+  public async getContentTypeUsages(query: Record<string, string>) {
```

Comment:
We can reuse he backend query type with something like:

```suggestion
  private async getWithQuery<Query, ResponseSchema>(
      url: string,
      query: Query
  ): Promise<APIResponse<ResponseSchema>> {
    const params = {} as Record<string, string>;

    for (let [key, value] of Object.entries(query)) {
      params[key] = value.toString();
    }
    
    return await this.get<ResponseSchema>(url, params);
  }

  public async getContentTypeUsages(query: Partial<GetContentUsagesQuery>) {
    return this.getWithQuery<Partial<GetContentUsagesQuery>, ContentUsageDto[]>(
      `${this.endpoints.contentUsages}`,
        query
    );
  }
```